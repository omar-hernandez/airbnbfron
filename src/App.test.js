// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });

import React, {Component} from 'react';
import {shallow} from 'enzyme';

import App from './App';

describe("App",()=>{

    it("Debe renderear correctamente", ()=>{
        const component = shallow(<App/>)
        expect(component).toMatchSnapshot();

    })
}) 