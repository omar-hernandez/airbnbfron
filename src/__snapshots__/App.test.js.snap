// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Debe renderear correctamente 1`] = `
<div>
  <Routes />
</div>
`;
